packages/date-picker/src/DatePicker/DatePicker.tsx

```diff
@@ -8,14 +8,18 @@ import { FieldMessageStatus } from "@kaizen/draft-form"
 import { calculateDisabledDays } from "../utils/calculateDisabledDays"
 import { isInvalidDate } from "../utils/isInvalidDate"
 import { isDisabledDate } from "../utils/isDisabledDate"
-import { setFocusInCalendar } from "../utils/setFocusInCalendar"
+import { setFocusInCalendar } from "../_primitives/Calendar/utils/setFocusInCalendar"
```

Comment:
Oh no :( I forgot this was used here. Hmm... maybe it belongs back in the root `utils` after all. Sorry! I know I told you to move it earlier 😅